app/src/components/nanomine/PageHeader.vue

```diff
@@ -51,9 +51,9 @@
                             <div class="nav_menu--container">
                                 <a class="u--default-size nav_menu--handler" href="#">About</a>
                                 <div class="nav_menu--siblings">
-                                    <router-link to="/about" class="nav_menu--siblings-lists"><a>About Us</a></router-link>
-                                    <router-link to="/how" class="nav_menu--siblings-lists"><a>How To</a></router-link>
-                                    <router-link to="/news" class="nav_menu--siblings-lists"><a>Research + News</a></router-link>
+                                    <router-link to="about" class="nav_menu--siblings-lists"><a>About Us</a></router-link>
+                                    <router-link to="how" class="nav_menu--siblings-lists"><a>How To</a></router-link>
+                                    <router-link to="news" class="nav_menu--siblings-lists"><a>Research + News</a></router-link>
                                 </div>
```

Comment:
For this I was thinking `to="/nm/about"` etc since I'm not sure if the router-link element can interpret nesting the same way, but if this also works than that is good!

Reply:
Also would you mind checking if the router links to these in components/Drawer.vue are working? I think when I wrote it I also had the routes as /about, /how, etc instead of /nm/about etc

Reply:
It loads up correctly when I run it, at least. The `to` prop uses `router.push()` [internally](https://router.vuejs.org/guide/essentials/navigation.html), and the way [nested routes](https://router.vuejs.org/guide/essentials/nested-routes.html) are defined implies that it should interpret it as a child route. So long as the header remains a `nanomine` component, it should be fine.

Reply:
Checked the Drawer as well, fixed in c8cc8e7.